content/docs/2.0/concepts/scaling-jobs.md

```diff
@@ -35,11 +36,100 @@ spec:
     template:
       # describes the [job template](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
   pollingInterval: 30  # Optional. Default: 30 seconds
-  cooldownPeriod:  300 # Optional. Default: 300 seconds
-  minReplicaCount: 0   # Optional. Default: 0
+  successfulJobsHistoryLimit: 5 # Optional. Default: 100. How many completed jobs should be kept.
+  failedJobsHistoryLimit: 5 # Optional. Default: 100. How many failed jobs should be kept.
   maxReplicaCount: 100 # Optional. Default: 100
   triggers:
   # {list of triggers to create jobs}
 ```
 
 You can find all supported triggers [here](../scalers).
+
+## Details
+
+```yaml
+  jobTargetRef:
+    parallelism: 1 # [max number of desired pods](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#controlling-parallelism)
+    completions: 1 # [desired number of successfully finished pods](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#controlling-parallelism)
+    activeDeadlineSeconds: 600 # Specifies the duration in seconds relative to the startTime that the job may be active before the system tries to terminate it; value must be positive integer
+    backoffLimit: 6 # Specifies the number of retries before marking this job failed. Defaults to 6
+```
+
+The optional configuration parameter. Currently not implemented. It is going to be supported. 
+
+```yaml
+  pollingInterval: 30  # Optional. Default: 30 seconds
+```
+
+This is the interval to check each trigger on. By default, KEDA will check each trigger source on every ScaledJob every 30 seconds.
+
+
+```yaml
+  successfulJobsHistoryLimit: 5 # Optional. Default: 100. How many completed jobs should be kept.
+  failedJobsHistoryLimit: 5 # Optional. Default: 100. How many failed jobs should be kept.
+```
+
+The `successfulJobsHistoryLimit` and `failedJobsHistoryLimit` fields are optional. These fields specify how many completed and failed jobs should be kept. By default, they are set to 100. This concept is similar to [Jobs Histry Limits](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#jobs-history-limits). The actual number of jobs could exceed the limit in a short time. However, it is going to resolve in the cleanup period. Currently, the cleanup period is the same as the Polling interval.
```

Comment:
```suggestion
The `successfulJobsHistoryLimit` and `failedJobsHistoryLimit` fields are optional. These fields specify how many completed and failed jobs should be kept. By default, they are set to 100.

This concept is similar to [Jobs History Limits](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#jobs-history-limits). The actual number of jobs could exceed the limit in a short time. However, it is going to resolve in the cleanup period. Currently, the cleanup period is the same as the Polling interval.
```

Reply:
However, it's still not sure what you mean with keep. Just resources that are in the cluster but completed?

Reply:
As you've noticed; I'm a noob to jobs so we should extend it a bit for new users who are noobs like me

Reply:
Hi @tomkerkhove 
If you run the ScaledJob, It create Jobs. That includes pods. Scaled Job create a Job. Then the Job finish running then complete or failure. 
The problem is, after the Job is completed, or failed, the job object still remains.  That could eat a lot of resources. `successfulJobsHistoryLimit` config will remove the job objects once it exceeds the number.  So that the the number of the job object will be the limited. So It doesn't consume a lot of resources. `Keep` means `preserve the job object`.  TBH, I'm not good at English. So any recommendation of wording is welcome! I appreciate you!

Reply:
Makes sense. What about this then?
```suggestion
The `successfulJobsHistoryLimit` and `failedJobsHistoryLimit` fields are optional. These fields specify how many completed and failed jobs should be kept. By default, they are set to 100.

This concept is similar to [Jobs History Limits](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#jobs-history-limits) allowing you to learn what the outcome of your jobs are.

The actual number of jobs could exceed the limit in a short time. However, it is going to resolve in the cleanup period. Currently, the cleanup period is the same as the Polling interval.
```